src/migrations/20221107121635-move-variants-to-per-environment.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+exports.up = function (db, callback) {
```

Comment:
I think we need to move this to be after the change request migrations so that it actually runs

Reply:
Lemme check that quick...

Reply:
Nevermind, Knex is smart enough to make sure this goes in